db/compaction/compaction_job.cc

```diff
@@ -614,14 +618,27 @@ Status CompactionJob::Run() {
   }
 
   IOStatus io_s;
+  for (const auto& state : compact_->sub_compact_states) {
```

Comment:
Do we need to duplicate this loop here? If the ```io_status``` for any of the ```sub_compact_states``` was non-ok, it would be reflected in their ```status``` right? So can we update ```io_s``` right after line 615?

Reply:
@anand1976 Yes, this loop is unnecessary. Will combine with the previous loop to get io_s.

Reply:
@anand1976 Updated